ERCS/erc-7644.md

```diff
@@ -0,0 +1,200 @@
+---
+eip: 7644
+title: ERC-721 Name Registry Extension
+description: Add a time-limited unique naming registry mechanism to ERC-721 tokens.
```

Comment:
Decent description&mdash;it does introduce more information than the title&mdash;but my concern here is that it doesn't really explain what a "naming registry" actually is.